components/backdrop/backdrop.js

```diff
@@ -162,20 +162,12 @@ function hideAccessible(target) {
 			if (path.indexOf(child) !== -1) continue;
 			if (child.hasAttribute(BACKDROP_HIDDEN)) continue;
 
-			const role = child.getAttribute('role');
-			if (role) child.setAttribute(BACKDROP_ROLE, role);
-			child.setAttribute('role', 'presentation');
-
-			if (child.nodeName === 'FORM' || child.nodeName === 'A') {
-				const ariaHidden = child.getAttribute('aria-hidden');
-				if (ariaHidden) child.setAttribute(BACKDROP_ARIA_HIDDEN, ariaHidden);
-				child.setAttribute('aria-hidden', 'true');
-			}
+			const ariaHidden = child.getAttribute('aria-hidden');
+			if (ariaHidden) child.setAttribute(BACKDROP_ARIA_HIDDEN, ariaHidden);
+			child.setAttribute('aria-hidden', 'true');
```

Comment:
Note that we have duplicate-ish code [in MVC](https://search.d2l.dev/xref/lms/lp/framework/web/D2L.LP.Web.UI/Desktop/Controls/Shim/Shim.js?r=d3403c8c#157) (which legacy actually thankfully uses too). Curious if this bug also happens there? If so, we'll want to apply a similar fix.

Reply:
Thanks, I'll have a look into that

Reply:
Doesn't appear to affect MVC Dialogs. Confirmed with Nathan that all reports have been related to creating/editing a quiz. I can't even get the LMS to crash in other areas that use the d2l-dialog.

Reply:
It might be a good idea anyway to update that code to be consistent with what we have here but we probably don't need to hotfix it

Reply:
I was able to reproduce the issue on a web component dialog from the top right ellipsis in Quick Eval, but yeh mostly just reproducable on FACE pages, with Android Chrome. And could not get it to fail with legacy dialogs even on FACE pages.

Reply:
So bizarre! Reminds me of like IE6 memory leak issues from the 2000s haha.